src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS = 'book-store//categories/STATUS';
+const INITIAL_STATE = [];
+
+export const checkStatus = (id) => ({
+  type: STATUS,
+  obj: id,
+});
+
+const reducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case STATUS:
+      return { status: 'Pending' };
```

Comment:
- [ ] **[OPTIONAL]** In case action.type is `STATUS` the function returns a string "Under construction" instead of `Pending`. Kindly fix it.